wcivf/apps/people/templates/people/includes/intros/_constituency.html

```diff
@@ -1,13 +1,13 @@
 {% extends "people/includes/intros/base.html" %}
-
+{% load i18n %}
 {% block intro_line %}
-    {{ person.name }} {{ verb }} the
-    <a href="{{ candidacy.party.get_absolute_url }}"
-        title="{{ candidacy.party_name }}">{{ candidacy.party_name }}</a>
-    candidate for <a
-        href="{{ candidacy.post_election.get_absolute_url }}">
-        {{ candidacy.post.label }}</a> constituency in the <a
-            href="{{ candidacy.election.get_absolute_url }}"
-            title="{{ candidacy.election.name }}">
-            {{ candidacy.election.name_without_brackets }}</a>.
+    {% blocktrans with name=person.name%}{{ name }} {{ verb }} the{% endblocktrans %}
+    <a href="{{ candidacy.party.get_absolute_url }}"title="{{ party }}">{{ party }}</a>
+    {% blocktrans with post_label=candidacy.post.label election=candidacy.election.name with election_name=candidacy.election.name_without_brackets %} candidate for
+        <a href="{{ candidacy.post_election.get_absolute_url }}">{{ post_label }}</a> constituency in the
+        <a href="{{ candidacy.election.get_absolute_url }}" title="{{ election }}">{{ election_name }}</a>
+    {% endblocktrans %}.
```

Comment:
I think you need to move the fullstop to after the closing `</a>` tag rather than after the `endblocktrans` as by having it on a new line it adds a whitespace (which is why one of tests is failing along with the above)